packages/calcite-components/src/components/action-pad/readme.md

```diff
@@ -1,6 +1,8 @@
 # calcite-action-pad
 
-The `calcite-action-pad` component is made up of `calcite-action`s in the form of clickable icons. This does not have an expandable option and is a smaller and simpler component than `calcite-action-bar`.
+The `calcite-action-pad` component is made up of `calcite-action`s in the form of clickable icons. Action Pad has rounded corners and box-shadows and should be positioned above other content.
```

Comment:
Same with Action Bar, I think the "in the form of clickable icons" is inaccurate. This can just be "a container for Action or Action Group components).